skymp5-client/src/modelSource/remoteServer.ts

```diff
@@ -268,6 +268,7 @@ export class RemoteServer implements MsgHandler, ModelSource, SendTarget {
               refr,
               !!msg.props['isHarvested'],
             );
+            ModelApplyUtils.applyModelIsDisabled(refr, !!msg.props['disabled']);
```

Comment:
better not call anything if `disabled` not found in props.

skyrim platform api is slow